themes/cloud-native-azure.omp.json

```diff
@@ -0,0 +1,158 @@
+{
+  "$schema": "https://raw.githubusercontent.com/JanDeDobbeleer/oh-my-posh/main/themes/schema.json",
+  "blocks": [
+    {
+      "alignment": "left",
+      "horizontal_offset": 0,
+      "segments": [
+        {
+          "background": "#c386f1",
+          "foreground": "#ffffff",
+          "leading_diamond": "\ue0b6",
+          "style": "diamond",
+          "trailing_diamond": "\ue0b0",
+          "type": "session"
+        },
+        {
+          "background": "#ff479c",
+          "foreground": "#ffffff",
+          "invert_powerline": false,
+          "powerline_symbol": "\ue0b0",
+          "properties": {
+            "prefix": " \uf07b  ",
+            "style": "folder"
+          },
+          "style": "powerline",
+          "type": "path"
+        },
+        {
+          "background": "#fffb38",
+          "foreground": "#193549",
+          "powerline_symbol": "\ue0b0",
+          "properties": {
+            "display_stash_count": true,
+            "display_upstream_icon": true,
+            "prefix": " \uf7a1  "
+          },
+          "style": "powerline",
+          "type": "git"
+        },
+        {
+          "background": "#2e9599",
+          "foreground": "#ffffff",
+          "leading_diamond": "<transparent,background>\ue0b0</>",
+          "properties": {
+            "always_enabled": true,
+            "color_background": true,
+            "display_exit_code": false,
+            "error_color": "#f1184c",
+            "prefix": " \ue23a "
+          },
+          "style": "diamond",
+          "trailing_diamond": "\ue0b4",
+          "type": "exit"
+        }
+      ],
+      "type": "prompt",
+      "vertical_offset": 0
+    },
+    {
+      "alignment": "right",
+      "horizontal_offset": 0,
+      "segments": [
+        {
+          "background": "#0077c2",
+          "foreground": "#ffffff",
+          "properties": {
+            "postfix": " <transparent,#0077c2>\ue0b2</>",
+            "prefix": "<#0077c2,transparent>\ue0b6</> \uf489 "
+          },
+          "style": "plain",
+          "type": "shell"
+        },
+        {
+          "background": "#f36943",
+          "foreground": "#ffffff",
+          "invert_powerline": true,
+          "powerline_symbol": "",
+          "properties": {
+            "charged_color": "#4caf50",
```

Comment:
this is no longer maintained and needs to be swapped to `background_templates` se the [docs](https://ohmyposh.dev/docs/battery#sample-configuration) for an example